### ISSUE TITLE
Error in test my own data *** Aborted at 1473133728 (unix time) try "date -d @1473133728" if you are using GNU date ***

### ISSUE THREAD

**Issue by bridge1118** [BVLC/caffe#4686]
I compile caffe and there are no warning even when i run 'make test', 'make runtest'
But when i ran my own data with 'bvlc_reference_caffenet' model i got the follow error warning
please help me!

GPU: Nvidia Quadro m4000
CUDA: Cuda-8.0
OS: ubuntu 16.04.1

I0906 11:48:42.870697  8068 solver.cpp:228] Iteration 60, loss = -nan
I0906 11:48:42.870822  8068 solver.cpp:244]     Train net output #0: loss = -nan (* 1 = -nan loss)
I0906 11:48:42.870832  8068 sgd_solver.cpp:106] Iteration 60, lr = 1.3
I0906 11:48:46.516229  8068 solver.cpp:454] Snapshotting to binary proto file models/mytrain_iter_61.caffemodel
*** Aborted at 1473133728 (unix time) try "date -d @1473133728" if you are using GNU date ***
PC: @     0x7ff574237c6c caffe::PoolingParameter::SerializeWithCachedSizesToArray()
*** Error in `/home/ful6ru04/caffe-master/tools/caffe': malloc(): memory corruption: 0x000000001773b870 ***
======= Backtrace: =========
/lib/x86_64-linux-gnu/libc.so.6(+0x77725)[0x7ff5721ff725]
/lib/x86_64-linux-gnu/libc.so.6(+0x819be)[0x7ff5722099be]
/lib/x86_64-linux-gnu/libc.so.6(__libc_malloc+0x54)[0x7ff57220b5a4]
/lib/x86_64-linux-gnu/libc.so.6(+0x6dc8d)[0x7ff5721f5c8d]
/usr/lib/x86_64-linux-gnu/libGL.so.1(+0xabd07)[0x7ff563b6ed07]
======= Memory map: ========
00400000-00419000 r-xp 00000000 08:02 1183432                            /home/ful6ru04/caffe-master/tools/caffe
00619000-0061a000 r--p 00019000 08:02 1183432                            /home/ful6ru04/caffe-master/tools/caffe
0061a000-0061b000 rw-p 0001a000 08:02 1183432                            /home/ful6ru04/caffe-master/tools/caffe
014ac000-1cae2000 rw-p 00000000 00:00 0                                  [heap]
200000000-200100000 rw-s 7b3716000 00:06 599                             /dev/nvidiactl
200100000-204300000 ---p 00000000 00:00 0
.
.
.
(i skip paste the rest content)

**Comment by shelhamer** [comment]
Please ask installation questions on the mailing list. This is an unusual crash and suggests an issue with your system, since this kind of memory corruption error is not usually encountered.

From https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md:

> _Please do not post usage, installation, or modeling questions, or other requests for help to Issues._
Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.
